.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+
+-   repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+    -   id: black
+        args: [--line-length=88]
```

Comment:
You probably want to switch to ruff :)

Reply:
I have heard of that! Tell me what you like about it!

Reply:
It is much faster and you can replace black, pylint and flake8 with just ruff and ruff-format :)

It has the same default settings as black